state/core_access.go

```diff
@@ -136,12 +138,17 @@ func (ca *CoreAccessor) Start(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("querying minimum gas price: %w", err)
 	}
+
+	err = ca.estimator.Start(ctx)
+	if err != nil {
+		log.Warn("state: failed to connect to estimator endpoint", "err", err)
+	}
```

Comment:
should we `return err` here? if no - comment why it's ok to skip it.

Reply:
we should actually return an error here bc the only time err is returned from estimator.Start is if grpc conn fails to be created which is an actual err.

Reply:
Done